site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -51,57 +54,63 @@ a list containing dictionaries of type [Ruleset][3], as shown below.
 }
 ```
 
-### Rule Resources {: #manifest-rule-resources }
+## Concepts and Usage {: #concepts-and-usage }
 
-Rules are specified in JSON files referenced under the `"rule_resources"` manifest key.
+To use declarative net requests specify one or more rulesets. A ruleset contains an array of rules. A single rule does one of the following:
 
-Each file should contain an array of [rules](#rules) as in the [example](#example).
+-  Block a network request.
+-  Prevent a request from getting blocked by negating any matching blocked rules.
+-  Redirect a network request.
+-  Modify request or response headers.
 
-**Note:** Errors and warnings about invalid static rules are only displayed for unpacked extensions.
-Invalid static rules in packed extensions are ignored. It's therefore important to verify that your
-static rulesets are valid by testing with an unpacked version of your extension.
+There are three types of rulesets.
 
-## Limits {: #limits }
+-  Static rules
```

Comment:
```suggestion
-  Static rulesets
```

etc.?

Reply:
Hmm, now it says "There are three types of rulesets." and then lists "static rules". That feels wrong too?

Reply:
I thought I had removed the word 'rules'.